repos/spack_repo/builtin/packages/mozjs/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.autotools import AutotoolsPackage
+
+from spack.package import *
+
+
+class Mozjs(AutotoolsPackage):
+    """SpiderMonkey is Mozilla’s JavaScript and WebAssembly Engine, used in Firefox, Servo and
+    various other projects. It is written in C++, Rust and JavaScript. You can embed it into C++
+    and Rust projects, and it can be run as a stand-alone shell."""
+
+    homepage = "https://spidermonkey.dev/"
+    url = "https://ftp.mozilla.org/pub/firefox/releases/128.1.0esr/source/firefox-128.1.0esr.source.tar.xz"
+
+    maintainers("KineticTheory")
+
+    license("Mozilla Public License, v. 2.0")
```

Comment:
should be
```suggestion
    license("MPL-2.0")
```
see https://spdx.dev/